mangobyte.py

```diff
@@ -100,13 +100,15 @@ async def on_ready():
 
 	general_cog = bot.get_cog("General")
 	audio_cog = bot.get_cog("Audio")
+	dota_cog = bot.get_cog("Dotabase")
 	artifact_cog = bot.get_cog("Artifact")
 	await artifact_cog.load_card_sets()
 	bot.help_command.cog = bot.get_cog("General")
 
 	periodic_tasks = [
 		general_cog.update_topgg,
-		general_cog.check_dota_patch
+		general_cog.check_dota_patch,
+                dota_cog.check_dota_blog
```

Comment:
This line has some spaces instead of tabs in it, could cause some issues with python i think. (ive had it complain to me before about files with differing indentations)